lib/screens/sentence_list_screen.dart

```diff
@@ -1 +1,75 @@
+import "package:flutter/material.dart";
+import "../services/sentence_service.dart";
+import "../models/sentence_model.dart";
+import "package:lingua_app/screens/sentence_item.dart";
 
+class SentenceListScreen extends StatefulWidget {
+  @override
+  _SentenceListScreenState createState() => _SentenceListScreenState();
+}
+
+class _SentenceListScreenState extends State<SentenceListScreen> {
+
+  Future<List<Sentence>> sentences;
+
+  Widget _errorWidget(String message){
+    return Center(
+      child: Column(
+        children: <Widget>[
+          Icon(Icons.block, size: 64.0),
+          Text("$message", style: TextStyle(color: Colors.red[400]))
+        ],
+      )
+    );
+  }
+
+  Widget _loadSentencesWidget() {
+    return FutureBuilder(
+        future: fetchSentences(),
+        builder: (context, snapshot) {
+          if (snapshot.connectionState == ConnectionState.none) {
+            return _errorWidget("No connection to API");
```

Comment:
can we be more specific about this? is it a server error or just that the user does not have internet connection?